### PR TITLE
function for user location

### PR DESCRIPTION
**Pull request by thakaresakshi121**
### What do you want to achieve?
- get location of user

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- 

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/kommunicate.js [hunk 1]
```diff
@@ -916,4 +916,31 @@ $applozic.extend(true, Kommunicate, {
...
+        }
+      
+        const getCurrentPosition = () => 
+          new Promise((resolve, reject) => 
+            navigator.geolocation.getCurrentPosition(resolve, reject)
+          );
+      
+        try {
+          const position = await getCurrentPosition();
+          const response = await fetch(`https://api.geoapify.com/v1/geocode/reverse?lat=${position.coords.latitude}&lon=${position.coords.longitude}&format=json&apiKey=8ce8e23a726b474cb13a056bd31dd070`);
```
Comment (devashishmamgain) [inline webplugin/js/app/kommunicate.js:932]: Pls use Google maps, remove geoapify

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/kommunicate.js [hunk 2]
```diff
@@ -916,4 +916,31 @@ $applozic.extend(true, Kommunicate, {
...
         }
     },
+    getUserLocation:async function (){
+        if (!("geolocation" in navigator)) {
+          console.error("Geolocation is not supported by this browser.");
+          return "Geolocation not supported";
+        }
+      
+        const getCurrentPosition = () => 
+          new Promise((resolve, reject) =>
```
Comment (panwaranuj01) [inline webplugin/js/app/kommunicate.js:926]: We already have the `latitude` and `longitude` global variables use those variables.

File: webplugin/js/app/kommunicate.js [hunk 3]
```diff
@@ -916,4 +916,31 @@ $applozic.extend(true, Kommunicate, {
             }
         }
     },
+    getUserLocation:async function (){
```
Comment (panwaranuj01) [inline webplugin/js/app/kommunicate.js:919]: put this function in mckMapService

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]

**Comment by panwaranuj01** [comment]
@thakaresakshi121 Base should `development` branch

File: webplugin/js/app/kommunicate.js (hunk 1 above)
Comment (thakaresakshi121) [reply webplugin/js/app/kommunicate.js]: updated  google map API

**Comment by thakaresakshi121** [comment]
raised Different PR (#1190 ) hence not needed

**Pull request closed without merging**